content/en/docs/reference/config/config-status/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Configuration Status Field
+description: Describes the role of the `status` field in configuration workflow.
+weight: 21
+---
+
+{{< warning >}}
+This feature is in the Alpha stage, see
+[Istio Feature Status](/about/feature-stages/). Your feedback is welcome in the
+[Istio User Experience discussion](https://discuss.istio.io/c/UX/23). Currently,
+this feature is tested only for single, low volume clusters with a single
+control plane revision.
+{{< /warning >}}
+
+Istio 1.6 and later provides information about the propagation of configuration
+changes through the mesh, using the `status` field in a resource's
+[`CustomResourceDefinition`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(CRD).
+
+The `status` field contains the state of a resource's configuration with various
+informational messages, including:
+
+* The resource's readiness.
+* How many data plane instances are associated with it.
+* Information for the output of tools, such as `istioctl analyze`.
+
+For example, the `kubectl wait` command monitors the `status` field to determine
+whether to unblock configuration and resume. For more information, see
+[Wait for Resource Status to Apply Configuration](/docs/ops/configuration/mesh/config-resource-ready/).
+
+## View the `status` field
+
+To view the contents of the `status` field, use the following command:
+
+{{< text bash >}}
+$ kubectl get virtualservice service-name
+{{< /text >}}
+
+In the output, the `status` field contains several child fields with details
+about the process of propagating configuration changes through the mesh. A
+child field for the condition status, also named `status`, has a value of `True`
```

Comment:
Here it should say describe the `conditions` field first. What is that, and why are there many? One for each what?

Reply:
"Conditions represent possible states of the resource.  In Istio 1.6, we are introducing two condition types, Reconciled and PassedAnalysis.  When the condition of type Reconciled has a status of false, that means that this resource is still in the progress of being distributed to all proxies.  When the condition of type PassedAnalysis has a status of false, this means that istio's background analyzers have detected a problem with your config, which will be detailed in the `validationMessages` field.

Reply:
On second thought, perhaps just remove the references to True and False up here, and leave the description of the Condition field.  You explain true and false very thoroughly later on, I think.

Reply:
fixed

Reply:
@adammil2000 Mitch's description, above, is much more clear than what's written in the following section. See my suggested change in the following section.